components/tidb_query_executors/src/index_scan_executor.rs

```diff
@@ -854,13 +903,20 @@ mod tests {
                 ci.set_pk_handle(true);
                 ci
             },
+            {
+                let mut ci = ColumnInfo::default();
+                ci.as_mut_accessor().set_tp(FieldTypeTp::LongLong);
+                ci.set_column_id(table::EXTRA_PHYSICAL_TABLE_ID_COL_ID);
+                ci
+            },
```

Comment:
Could you also have a test to ensure that, when PhysicalTableId column is NOT pushed down, the code works?